src/main/java/sx/blah/discord/handle/audit/entry/change/ChangeKey.java

```diff
@@ -0,0 +1,92 @@
+/*
+ *     This file is part of Discord4J.
+ *
+ *     Discord4J is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU Lesser General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     (at your option) any later version.
+ *
+ *     Discord4J is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU Lesser General Public License for more details.
+ *
+ *     You should have received a copy of the GNU Lesser General Public License
+ *     along with Discord4J.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package sx.blah.discord.handle.audit.entry.change;
+
+import sx.blah.discord.Discord4J;
+import sx.blah.discord.handle.obj.IChannel;
+import sx.blah.discord.handle.obj.IRole;
+import sx.blah.discord.util.LogMarkers;
+
+/**
+ * The keys used in an {@link ChangeMap}. Use these with
+ * {@link sx.blah.discord.handle.audit.entry.AuditLogEntry#getChangeByKey(ChangeKey)}.
+ *
+ * @param <T> The type of the value associated with the key.
+ */
+public final class ChangeKey<T> {
+	public static final ChangeKey<String> NAME = newKey();
+	public static final ChangeKey<String> ICON_HASH = newKey();
+	public static final ChangeKey<String> SPLASH_HASH = newKey();
+	public static final ChangeKey<Long> OWNER_ID = newKey();
+	public static final ChangeKey<String> REGION = newKey();
+	public static final ChangeKey<Long> AFK_CHANNEL_ID = newKey();
+	public static final ChangeKey<Integer> AFK_TIMEOUT = newKey();
+	public static final ChangeKey<Integer> MFA_LEVEL = newKey();
+	public static final ChangeKey<Integer> VERIFICATION_LEVEL = newKey();
+	public static final ChangeKey<Integer> EXPLICIT_CONTENT_FILTER = newKey();
+	public static final ChangeKey<Integer> DEFAULT_MESSAGE_NOTIFICATIONS = newKey();
+	public static final ChangeKey<String> VANITY_URL_CODE = newKey();
+	public static final ChangeKey<IRole[]> $ADD = newKey();
```

Comment:
I'm not a huge fan of making small changes like that for the sake of holding the user's hand. It makes for uglier internal code (we have to manually handle the raw name of the string) as well as introduces the possibility of breaking changes should Discord ever decide to change the name.